test/testing_utils_test.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:atom_dartlang/impl/testing_utils.dart';
+import 'package:test/test.dart';
+
+main() => defineTests();
+
+defineTests() {
+  group('test_utils', () {
+    test('getPossibleTestPaths', () {
+      expect(getPossibleTestPaths('bin/foo.dart'), equals([]));
```

Comment:
I recommend using join rather than '/' so that tests will use Windows paths on Windows. Alternately create a utility that will accept posix path and return os path so that all tests can be written with posix paths (easier to read than lots of join calls). Perhaps something like...
```
expect(getPossibleTestPaths(p('bin/foo.dart')), equals([]));
```
where `p(String posix)` is a local utility that converts a posix path to an os path.